app/controllers/FileReceivedController.scala

```diff
@@ -51,13 +53,22 @@ class FileReceivedController @Inject() (
         fileDetails =>
           (getContactEmails, fileDetails) match {
             case (Some(emails), Some(details)) =>
-              val time = details.submitted.format(timeFormatter).toLowerCase
-              val date = details.submitted.format(dateFormatter)
-              for {
-                updatedAnswers <- Future.fromTry(request.userAnswers.remove(UploadIDPage))
-                _              <- sessionRepository.set(updatedAnswers)
-              } yield Ok(view(details.messageRefId, time, date, emails.firstContact, emails.secondContact))
-            case _ => Future.successful(InternalServerError(errorView()))
+              if (details.reportType.contains(SingleOther)) {
+                logger.warn("FileReceivedController: Test data submitted but successful outcome")
+                Future.successful(InternalServerError(errorView()))
+              } else {
+                val detailsList =
```

Comment:
Can this be added to the FileReceivedViewModel

Reply:
I'm having to pass detailsList to the view so not sure how I'd go about it